File: Leetcode/Python/subtract_product_and_sum.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        product = 1
        sum = 0

        while True:
            if n == 0:
                break
            else:
                remainder = n%10
                product = product * remainder
                sum += remainder
                n = n//10

        return product - sum

File: Leetcode/CPP/subtract_product_and_sum.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int product = 1;
        int sum = 0;
        
        while(true){
            
            if (n == 0){
                break;
            }
            else{
                int remainder = n%10;
                product = product * remainder;
                sum = sum + remainder;
                n = n/10;
            }
        }
        
     return product - sum;   
    }
};
